microsoft/testsuites/xdp/pktgen.py

```diff
@@ -195,6 +198,12 @@ def _install_fedora(self) -> None:
                 "/all/Packages/k/kernel-modules-internal-"
                 f"{'.'.join(parts[0:3])}-{'.'.join(parts[3:-1])}.{parts[-1]}.rpm"
             ),
+            (
+                "http://20.151.28.143/openlogic/"
```

Comment:
Where is the IP from? Can we use a domain name? IP is not reliable.

Reply:
this is the only place I can find for kernel-modules-internal-5.4.155-200_ol008.el7.aarch64.rpm of centos 7.9 arm image, we can move all of them which we can find to our storage account

Reply:
Let's skip this kernel, if it's not an official place. It's not good to keep them in our storage account.

Reply:
Is it another thing? I'm ok to install in parallel. I mean, if the package is got from an IP address, we cannot use it.